sample/Pizzeria.Store.Web/Layout/MainLayout.razor

```diff
@@ -1,4 +1,5 @@
 ﻿@inherits LayoutComponentBase
+@rendermode InteractiveServer

```

Comment:
We want client rendering, so that we can more easily transition to a Blazor WASM app when Aspire supports it, not interactive rendering